packages/core/src/components/region/RegionBar/RegionBar.tsx

```diff
@@ -73,8 +95,9 @@ function RegionBar({
       onButtonClick={openModal}
       postalCode={postalCode}
       city={textToTitleCase(city ?? '')}
-      shouldDisplayPostalCode={shouldDisplayPostalCode}
+      shouldDisplayPostalCode={true}
```

Comment:
I haven't removed this prop yet from `RegionBar`. But I've set is as true for now because I've discussed with Renatinha and decided to always show city and postalCode when available. (But she was going to double check this decision with Bia)